### ISSUE TITLE
Cannot access class CSVFormat on writing into csv file

### ISSUE THREAD

**Issue by Tapchicoma** [Kotlin/dataframe#75]
Project declares only following dependency to this project:
```
implementation("org.jetbrains.kotlinx:dataframe:0.8.0-rc-2")
```

When I try to compile a file that has following line:
```kt
dataFrame.writeCSV(benchmarkCsvFile)
```
Compilation fails with following message:
```
Cannot access class 'org.apache.commons.csv.CSVFormat'. Check your module classpath for missing or conflicting dependencies
```

**Comment by Tapchicoma** [comment]
Adding `compileOnly("org.apache.commons:commons-csv:1.8")` dependency fixes compilation issue.

**Comment by ileasile** [comment]
It was fixed two days ago: https://github.com/Kotlin/dataframe/commit/5a0668c4bc28533c90ef8baf1e12ea344dd29c14
Should be present in the nearest upcoming release

**Comment by nikitinas** [comment]
Fixed in `0.8.0-rc-3`
